<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>100</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>100</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="6,2,2,2,0,0,2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="titulo">
       <property name="styleSheet">
        <string notr="true">font: 24pt &quot;Arial Rounded MT Bold&quot;;
color: #957F5F;
</string>
       </property>
       <property name="text">
        <string>Vista de Formulario</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="combobox_colores">
       <property name="minimumSize">
        <size>
         <width>9</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
	width: 30px;
	height: 30px;
	border-radius: 15px;
	border: none;
	background-color: rgb(185, 185, 185);
	font: 75 12pt &quot;MS Sans Serif&quot;;

}
QComboBox::drop-down {
                subcontrol-origin: padding;
                subcontrol-position: top right;
                width: 0px;
                height: 0px;
                border-left-width: 0px;
                border-top-right-radius: 15px;
                border-bottom-right-radius: 15px;
            }
QComboBox QAbstractItemView {
    min-width: 250px;
	border-radius: 6px;
	background-color: rgb(255, 255, 255);
    padding: 10px; /* Relleno interno de 5px */
    outline: none; /* Eliminar el resaltado cuando se selecciona un elemento */
}

QListView::item {
    height: 30px;
}



</string>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Ninguno</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>resources/icons/circle-gray.png</normaloff>resources/icons/circle-gray.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tr√°mites y pagos finalizados</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>resources/icons/circle-green.png</normaloff>resources/icons/circle-green.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tramites pendientes</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>resources/icons/circle-yellow.png</normaloff>resources/icons/circle-yellow.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pagos pendientes</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>resources/icons/circle-red.png</normaloff>resources/icons/circle-red.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_pdf">
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius: 8px;
background-color: #957F5F;
color: white;
font-size: 16px;
padding: 10px 30px;
}

QPushButton:hover {
	background-color: rgb(116, 91, 47);
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
    border-style: inset;
    background-color: rgb(103, 80, 41);
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Crear PDF</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_error">
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Arial&quot;;
color: #8fc26b;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_aviso_vencimiento">
     <property name="styleSheet">
      <string notr="true">font: 75 18pt;
color: #666666;
font-weight: 700;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollArea{
border: none;
background: white;
}
QScrollBar:vertical {
	border: none;
    background: white;
    width: 14px;
    margin: 15px 0 15px 0;
	border-radius: 0px;
 }

/*  HANDLE BAR VERTICAL */
QScrollBar::handle:vertical {	
	background-color: #CCCCCC;
	min-height: 30px;
	border-radius: 7px;
}
QScrollBar::handle:vertical:hover{	
	background-color: #666666;
}
QScrollBar::handle:vertical:pressed {	
	background-color: #666666;
}

/* BTN TOP - SCROLLBAR */
QScrollBar::sub-line:vertical {
	border: none;
	background-color: white;
	height: 15px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical:hover {	
	background-color: white;
}
QScrollBar::sub-line:vertical:pressed {	
	background-color: white
}

/* BTN BOTTOM - SCROLLBAR */
QScrollBar::add-line:vertical {
	border: none;
	background-color: white
	height: 15px;
	border-bottom-left-radius: 7px;
	border-bottom-right-radius: 7px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}
QScrollBar::add-line:vertical:hover {	
	background-color: white;
}
QScrollBar::add-line:vertical:pressed {	
	background-color: white;
}

/* RESET ARROW */
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	background: none;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
}
</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1081</width>
        <height>18</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>25</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_confirmar">
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius: 8px;
background-color: #957F5F;
color: white;
font-size: 16px;
padding: 10px 30px;
}

QPushButton:hover {
	background-color: rgb(116, 91, 47);
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
    border-style: inset;
    background-color: rgb(103, 80, 41);
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Confirmar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_cancelar">
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius: 8px;
border: 1px solid #957F5F;
background-color: white;
color: #957F5F;
font-size: 16px;
padding: 10px 30px;
}

QPushButton:hover {
	background-color: rgb(116, 91, 47);
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
    border-style: inset;
    background-color: rgb(103, 80, 41);
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
